grails-app/views/inventoryItem/_showStockHistory.gsp

```diff
@@ -114,7 +118,40 @@
             $('.monthRow').hide().children().removeClass("active");
             $('.dataRow').hide();
           }
-        })
+        });
+
+        $('.showYear').on('click', function(event) {
+          var tr = $(this).closest('tr');
+          var td = $(this).closest('td');
+          var monthRows = getMonthRows(tr);
+          var isAnyHidden = true;
+
+          $.each(monthRows, function() {
+            if (isAnyHidden) {
```

Comment:
I don't get the point of it. Why this can't be in the next 'each' loop and why there is this if?